src/bindgen/ir/enumeration.rs

```diff
@@ -181,13 +181,13 @@ impl EnumVariant {
         }
     }
 
-    fn specialize(&self, generic_values: &[Type], mappings: &[(&Path, &Type)]) -> Self {
+    fn specialize(&self, generic_values: &[Type], mappings: &[(&Path, &Type)], mangling_separator: &Option<String>) -> Self {
```

Comment:
Passing an `Option<&str>` is a bit more idiomatic, is it much more difficult?

Alternatively if passing the whole `&Config` down here is sensible I think that'd be slightly preferable, but no big deal.